lib/rucio/rse/protocols/storm.py

```diff
@@ -0,0 +1,146 @@
+    """ Implementing access to RSEs using the local filesystem."""
+
+    def __init__(self, protocol_attr, rse_settings):
+        """ Initializes the object with information about the referred RSE.
+
+            :param props Properties derived from the RSE Repository
+        """
+        super(Default, self).__init__(protocol_attr, rse_settings)
+        self.attributes.pop('determinism_type', None)
+        self.files = []
+
+    def _get_path(self, scope, name):
+        """ Transforms the physical file name into the local URI in the referred RSE.
+            Suitable for sites implementoing the RUCIO naming convention.
+
+            :param name: filename
+            :param scope: scope
+
+            :returns: RSE specific URI of the physical file
+        """
+        return '%s/%s' % (scope, name)
+
+    def path2pfn(self, path):
+        """
+            Retruns a fully qualified PFN for the file referred by path.
+
+            :param path: The path to the file.
+
+            :returns: Fully qualified PFN.
+
+        """
+        return ''.join([self.rse['scheme'], '://%s' % self.rse['hostname'], path])
+
+    def exists(self, pfn):
+        """ Checks if the requested file is known by the referred RSE.
+
+            :param pfn Physical file name
+
+            :returns: True if the file exists, False if it doesn't
+
+            :raise  ServiceUnavailable
+        """
+        raise NotImplementedError
+
+    def connect(self):
+        """ Establishes the actual connection to the referred RSE.
+
+            :param credentials Provide all necessary information to establish a connection
+                to the referred storage system. Some is loaded from the repository inside the
+                RSE class and some must be provided specific for the SFTP protocol like
+                username, password, private_key, private_key_pass, port.
+                For details about possible additional parameters and details about their usage
+                see the pysftp.Connection() documentation.
+                NOTE: the host parametrer is overwritten with the value provided by the repository
+
+            :raise RSEAccessDenied
+        """
+        raise NotImplementedError
+
+    def close(self):
+        """ Closes the connection to RSE."""
+        raise NotImplementedError
+
+    def get(self, pfn, dest, transfer_timeout=None):
+        """ Provides access to files stored inside connected the RSE.
+
+            :param pfn Physical file name of requested file
+            :param dest Name and path of the files when stored at the client
+            :param transfer_timeout Transfer timeout (in seconds)
+
+            :raises DestinationNotAccessible, ServiceUnavailable, SourceNotFound
+         """
+
+        # retrieve the TURL from the webdav etag
+        cmd = 'davix-http --capath /cvmfs/atlas.cern.ch/repo/ATLASLocalRootBase/etc/grid-security-emi/certificates --cert $X509_USER_PROXY -X PROPFIND %s' % pfn
```

Comment:
this will only work for atlas. we can keep it like that for now, but if you find time, you can make it configurable with core.config